react/react1/week3/homework/shift-app/src/usercomponents/UserList.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import FancyBorder from "./FancyBorder";
+import SubmitShift from "./SubmitShift";
+import SubTitle from "./SubTitle";
+import Search from "./Search";
+import User from "./User";
+import Total from "./Total";
+const UserList = ({ fetchUrl }) => {
+  const [users, setUsers] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    setLoading(true);
+    const timeOut = setTimeout(getResult, 3000);
+    return () => clearTimeout(timeOut);
+  }, []);

```

Comment:
You dont need to set a timeout here. You can call getResult function directly here. Also getResult ils already including Loading statement.
```suggestion
  useEffect(() => {
    getResult()
  }, []);

```

Reply:
Thanks ,  I have just implemented the changes,  I was trying to show the loading text  a little longer with timeout.